util/branded.ts

```diff
@@ -1,9 +1,4 @@
-export type Branded<T, Brand extends PropertyKey> = T & { [_ in Brand]: true }
-
-declare const _hex: unique symbol
-export type Hex = Branded<string, typeof _hex>
-
-declare const _hash: unique symbol
-export type Hash = Branded<Uint8Array, typeof _hash>
-
-export type HexHash = Hex & Hash
+// TODO: delete these upon re-rpc merge
+export type Hex = string
```

Comment:
Why this, here?

Reply:
DNT started throwing an error regarding serialization of symbols `hex_` and `hash_` (even if we export them from the root mod). This was the fix.